external/avro/src/test/scala/org/apache/spark/sql/avro/AvroSuite.scala

```diff
@@ -492,17 +493,21 @@ abstract class AvroSuite
       df.write.format("avro").save(deflateDir)
       spark.conf.set(SQLConf.AVRO_COMPRESSION_CODEC.key, "snappy")
       df.write.format("avro").save(snappyDir)
+      spark.conf.set(SQLConf.AVRO_COMPRESSION_CODEC.key, "zstandard")
+      df.write.format("avro").save(zstandardDir)
```

Comment:
@dongjoon-hyun To `zstandard` codec.

Reply:
So, this is failing still?

Reply:
Yes.

Reply:
Thank you for confirming and sharing this a nice test case. :)

Reply:
https://github.com/apache/spark/runs/1993307091?check_suite_focus=true